### ISSUE TITLE
Issue with Accessibility and SEO

### ISSUE THREAD

**Issue by mateusoliveira43** [mateusoliveira43/mateusoliveira43.github.io#7]
**What is the issue?**
The website does not have a code check regarding Search Engine Optimization (SEO) and accessibility.

**What do you suggest?**
Configure an automated way of checking this issues ([link that can help](https://blog.codeminer42.com/a-short-guide-to-web-accessibility/)).
